src/components/Icon/Illustrations.ts

```diff
@@ -50,6 +52,9 @@ import MoneyBadge from '@assets/images/simple-illustrations/simple-illustration_
 import MoneyIntoWallet from '@assets/images/simple-illustrations/simple-illustration__moneyintowallet.svg';
 import MoneyWings from '@assets/images/simple-illustrations/simple-illustration__moneywings.svg';
 import OpenSafe from '@assets/images/simple-illustrations/simple-illustration__opensafe.svg';
+import PalmTree from '@assets/images/simple-illustrations/simple-illustration__palmtree.svg';
+import Profile from '@assets/images/simple-illustrations/simple-illustration__profile.svg';
+import QrCode from '@assets/images/simple-illustrations/simple-illustration__qr-code.svg';
```

Comment:
Naming styling, can be fixed in a follow up
```suggestion
import QRCode from '@assets/images/simple-illustrations/simple-illustration__qr-code.svg';
```

Reply:
Going to rename it [here](https://github.com/Expensify/App/pull/35672).